src/app/assets/languages/en.json

```diff
@@ -77,9 +77,9 @@
     "cache-filter-feature-label" : "Feature",
     "cache-filter-feature-bounded" : "Bounded",
     "cache-filter-feature-indexed" : "Indexed",
-    "cache-filter-feature-persistent" : "Persistent",
-    "cache-filter-feature-trans" : "Transactional",
-    "cache-filter-feature-secure" : "Secured",
+    "cache-filter-feature-persistent" : "Persistence",
+    "cache-filter-feature-trans" : "Transactions",
+    "cache-filter-feature-secure" : "Authorization",
```

Comment:
@dipguptaredhat Hey I tried to change the labels in the feature drop-down but it's still showing "Transactional" and I couldn't quite figure out why this doesn't take effect. Would you mind taking a look please? 

![image](https://user-images.githubusercontent.com/32749632/173820067-59341e7c-e042-45c1-ae3c-f1e031fbf175.png)